<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleItemEditDialog</class>
 <widget class="QDialog" name="RuleItemEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование элемента диагностического правила</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Симптом</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="m_symptomName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_chooseSymptom">
         <property name="toolTip">
          <string>Выбрать симптом</string>
         </property>
         <property name="type" stdset="0">
          <string>edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Значение</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QStackedWidget" name="m_stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="m_textValuePage">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_textValue"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_realValuePage">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="m_realValue"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="m_realValue2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_enumValuePage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="m_enumValue"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>P(S|D)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="m_probabilityWithDisease">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>P(S|¬D)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="m_probabilityWithoutDisease">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Оператор</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_operator"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>P(S|D) -- это вероятность появления симптома при наличии болезни.
P(S|¬D) -- это вероятность появления симптома при отсутствии болезни.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_chooseSymptom</tabstop>
  <tabstop>m_operator</tabstop>
  <tabstop>m_textValue</tabstop>
  <tabstop>m_realValue</tabstop>
  <tabstop>m_realValue2</tabstop>
  <tabstop>m_enumValue</tabstop>
  <tabstop>m_probabilityWithDisease</tabstop>
  <tabstop>m_probabilityWithoutDisease</tabstop>
  <tabstop>m_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
